### PR TITLE
[SDEM] Added Chien drag law

### PR DESCRIPTION
**Pull request by farrufat-cimne**
**Description**
Recovered Chien drag type from legacy repo.

### CODE REVIEW

**Comment by maceligueta** [comment]
Are there tests for the drag laws? We should keep the habit of writing tests for every new feature.

**Comment by farrufat-cimne** [comment]
@maceligueta Currently there no tests focused on testing the different drag laws.

File: applications/SwimmingDEMApplication/custom_python/add_custom_constitutive_laws_to_python.cpp [hunk 1]
```diff
@@ -134,6 +135,12 @@ void AddCustomConstitutiveLawsToPython(pybind11::module& m) {
         .def(py::init<Parameters&>())
         ;
 
+    py::class_<ChienDragLaw, ChienDragLaw::Pointer, BaseDragLawType>(m, "ChienDragLaw")
+        .def(py::init<>())
+        .def(py::init<Parameters&>())
```
Comment (philbucher) [inline applications/SwimmingDEMApplication/custom_python/add_custom_constitutive_laws_to_python.cpp]:
```suggestion
        .def(py::init<Parameters>())
```

`Parameters` should be passed by value

**Comment by farrufat-cimne** [comment]
added missing changes in headers and cpp to be consistent with pybind.

----------------------------------------------------------------------
Ran 13 tests in 2.717s

OK

**Comment by farrufat-cimne** [comment]
Shortened execution time.
Ran 1 test in 0.183s
OK

**Review by maceligueta — APPROVED** [review]

**Pull request merged**